docs/resources/cluster_rosa_classic.md

```diff
@@ -47,47 +47,47 @@ resource "rhcs_cluster_rosa_classic" "rosa_sts_cluster" {
 
 ### Required
 
-- `aws_account_id` (String) Identifier of the AWS account.
+- `aws_account_id` (String) Identifier of the AWS account.Once the resource was created, the attribute cannot be changed.
```

Comment:
```suggestion
- `aws_account_id` (String) Identifier of the AWS account.Once the resource is created, the attribute cannot be changed.
```

Not really sure if this is a better terminology or not. Might be nice to talk about it with other folks

Reply:
In addition, maybe better to write this once in const and than use it in all places

Reply:
I changed it to be `After the creation of the resource, it is not possible to update the attribute value.`
And it's a const string in common package